### PR TITLE
fix bug after commit 3dfa11b89e6a74e6857c11e3029b372c746d535f

### PR DESCRIPTION
**Pull request by leommxj**

### CODE REVIEW

File: apps/grgsm_decode [hunk 1]
```diff
@@ -350,16 +350,16 @@ if __name__ == '__main__':
...
                 arfcn = options.arfcn
-                fc = grgsm.arfcn.arfcn2downlink(arfcn, options.band)
+                fc = grgsm.arfcn.arfcn2downlink(arfcn)
         else:
             arfcn = options.arfcn
             for band in grgsm.arfcn.get_bands():
-                if grgsm.arfcn.is_valid_arfcn(arfcn, band):
-                    fc = grgsm.arfcn.arfcn2downlink(arfcn, band)
+                if grgsm.arfcn.is_valid_arfcn(arfcn):
+                    fc = grgsm.arfcn.arfcn2downlink(arfcn)
```
Comment (petterreinholdtsen) [inline apps/grgsm_decode]: Do you still need the for loop now that "band" is no longer needed?

File: apps/helpers/grgsm_capture [hunk 2]
```diff
@@ -139,11 +139,11 @@ class grgsm_capture(gr.top_block):
...
             self.gsm_receiver.set_cell_allocation([self.arfcn])
             if options.band:
-                new_freq = grgsm.arfcn.arfcn2downlink(self.arfcn, self.band)
+                new_freq = grgsm.arfcn.arfcn2downlink(self.arfcn)
             else:
                 for band in grgsm.arfcn.get_bands():
-                    if grgsm.arfcn.is_valid_arfcn(arfcn, band):
-                        new_freq = grgsm.arfcn.arfcn2downlink(arfcn, band)
+                    if grgsm.arfcn.is_valid_arfcn(arfcn):
+                        new_freq = grgsm.arfcn.arfcn2downlink(arfcn)
```
Comment (petterreinholdtsen) [inline apps/helpers/grgsm_capture]: Do you still need the for loop when "band" is no longer needed?

Comment (leommxj) [reply apps/helpers/grgsm_capture]: No,there is no need for the for loop.^_^
Yesterday,I was just trying to make it work,didn't think it through. 
Now i'm kind of thinking there is no need for option:'band' if option:'arfcn' is set.

**Comment by ptrkrysik** [comment]
Thanks guys for your work. I pushed the arfcn change definitely too fast. Probably adding more automatic tests could allow to catch such errors more easily. The corrections were only briefly checked by me so there might still be some bug hidden somewhere, but the most obvious bugs that prevent the programs from running for any combination of input parameters are probably fixed now.

**Pull request merged**